### DISCUSSION TITLE
Support Pyright's strict typing.

### DISCUSSION THREAD

**Discussion by aspizu** [zauberzeug/nicegui#762] [Ideas / Feature Requests]
### Description

Support pyright's strict typing mode. where it reports errors such as unknown type.

**Comment by falkoschindler** [comment]
Hello @aspizu! We appreciate your suggestion to enhance NiceGUI's type annotations. It's a valuable contribution to the project.

We warmly welcome pull requests, and even partial improvements to the type annotations are a fantastic way to contribute and become an official part of our community. If you're interested, feel free to submit your changes and we'll be happy to review and collaborate with you on refining the code.
